media-api/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -312,7 +312,10 @@ class ElasticSearch(
       case _ => List(imagesCurrentAlias)
     }
     val migrationAwareQuery = migrationStatus match {
-      case running: Running => filters.and(query, filters.mustNot(filters.term("esInfo.migration.migratedTo", running.migrationIndexName)))
+      case running: Running => filters.and(query, filters.and(
```

Comment:
should it be 
```suggestion
      case running: Running => filters.and(query, filters.or(
```

Reply:
I'm pretty sure `and` is right (but I'm having trouble working through booleans this morning) - I think `or` would mean we'd get the duplicates on all the non-syndication related images. written out in a pseudocode 

```
query && (!migratedTo && !syndicationRightsMissing)
vs
query && (!migratedTo || !syndicationRightsMissing)
```

Reply:
yeah checked on TEST - it works how we want with and, was just second guessing myself. im just waiting for `main` to redeploy to TEST to confirm that my replicating of the scenario was fine and that we see the duplicates.

Reply:
yep, i had replicated it correctly and this PR does indeed solve it, merging now...